src/tests/TaskList.test.tsx

```diff
@@ -0,0 +1,76 @@
+// src/tests/TaskList.test.tsx
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import TaskList from '../components/TaskList';
+
+describe('Task Management App', () => {
+    test('allows user to add a new task with category and priority', () => {
+        render(<TaskList />);
+
+        const taskInput = screen.getByPlaceholderText(/Task description/i);
+        const categoryInput = screen.getByPlaceholderText(/Task Category/i);
+        const prioritySelect = screen.getByDisplayValue('Low Priority');
+        const addButton = screen.getByRole('button', { name: /Add Task/i });
+
+        fireEvent.change(taskInput, { target: { value: 'New Task' } });
+        fireEvent.change(categoryInput, { target: { value: 'Work' } });
+        fireEvent.change(prioritySelect, { target: { value: 'High' } });
+        fireEvent.click(addButton);
+
+        // Verify the task text, category, and priority
+        expect(screen.getByText(/New Task/i)).toBeInTheDocument();
```

Comment:
expect/match for exact value including the casing

Reply:
fixed